cabal.project

```diff
@@ -9,7 +9,7 @@ package aeson
 source-repository-package
     type: git
     location: https://github.com/kadena-io/pact.git
-    tag: 9456a750f67b62d48bed5ef2e60fbc7e9d2a4396
+    tag: b5601c601c3e497c938f6d804eb21f9a3fb30a2a
```

Comment:
This is the commit from the PR. The merged commit is: 0915aff9eed762b2ee58c648d900bc619e6fff96